services/elasticache/pkg/resource/replication_group/custom_update_api.go

```diff
@@ -110,144 +180,184 @@ func (rm *resourceManager) updateReplicaCount(
 	return provideUpdatedResource(desired, resp.ReplicationGroup)
 }
 
-// newUpdate(ShardConfiguration)RequestPayload returns an SDK-specific struct for the HTTP request
-// payload of the Update API call for the resource
-func (rm *resourceManager) newUpdateShardConfigurationRequestPayload(
+func (rm *resourceManager) updateShardConfiguration(
+	ctx context.Context,
 	desired *resource,
 	latest *resource,
-) (*svcsdk.ModifyReplicationGroupShardConfigurationInput, error) {
-	res := &svcsdk.ModifyReplicationGroupShardConfigurationInput{}
+) (*resource, error) {
+	input, err := rm.newUpdateShardConfigurationRequestPayload(desired, latest)
+	if err != nil {
+		return nil, err
+	}
+	resp, respErr := rm.sdkapi.ModifyReplicationGroupShardConfigurationWithContext(ctx, input)
+	if respErr != nil {
+		return nil, respErr
+	}
+	return provideUpdatedResource(desired, resp.ReplicationGroup)
+}
+
+// new(IncreaseReplicaCount)RequestPayload returns an SDK-specific struct for the HTTP request
```

Comment:
Can you remove the parens around IncreaseReplicaCount above?

Reply:
Updated source file.